cl/users/templates/profile/settings.html

```diff
@@ -170,6 +171,19 @@
                     </p>
                 {% endif %}
             </div>
+
+	    <div class="form-group" >
+		<label for="id_sort_desc" >
+		    {{ profile_form.sort_desc }}
+		    Sort dockets in descending order</label>
+		{% if profile_form.sort_desc.errors %}
+		    <p class="help-block">
+			{% for error in profile_form.sort_desc.errors %}
+			    {{ error|escape }}
+			{% endfor %}
+		    </p>
+		{% endif %}
+	    </div>
```

Comment:
Could you clean up the indentation here a bit, please, so it's more like (or better than) the others (which I acknowledge are kind of weird)?

Reply:
I'm sorry, I'm looking at the block immediately above `wants_newsletter` and it looks to my blind eyes to be formatted exactly like I formatted the `sort_desc` block.  I don't know what you are seeing.

Reply:
Maybe your editor is lying to you? 

![image](https://user-images.githubusercontent.com/236970/229642183-3c4fa893-5ac6-47a3-9570-3629cab51c53.png)